README.md

```diff
@@ -98,10 +98,46 @@ A problem occurred while evaluating entry:
 
 #### Properties inheritance
 It might be useful to have properties files that can recursively include
-other properties files (specified via an `include` property).
+other properties files (specified via an `include` property with including file path as value).
 Inherited properties can be overridden by the including set, just redefine
 the property in the file and its value will be used instead of the one
-from the included set.
+from the included set, eg:
+
+Given a generic properties file `config.properties`:
+
+```properties
+foo=bar
+aKey=aValue
+```
+
+You might include previous one into a new properties file `debug.properties`:
+
+```properties
+include=/path/to/config.properties
+aNewKey=aNewValue
+aKey=overriddenPreviousValue
+```
+
+Then in your `build.gradle`:
+
+```gradle
+buildProperties {
+    secrets {
+        file rootProject.file('debug.properties')
+    }
+}
+
+...
+
+    defaultConfig {
```

Comment:
if including the indent, can you also add the `android` closure to show that this config lives there?